Shell/main.cpp

```diff
@@ -111,7 +111,7 @@ int main(int argc, char** argv)
     Core::ArgsParser parser;
     parser.add_option(command_to_run, "String to read commands from", "command-string", 'c', "command-string");
     parser.add_option(skip_rc_files, "Skip running shellrc files", "skip-shellrc", 0);
-    parser.add_option(format, "File to format", "format", 0, "file");
+    parser.add_option(format, "Format the given file into stdout and exit", "format", 0, "file");
```

Comment:
Hmm, wait, why is this better? This seems inconsistent with every other help string?

Reply:
Is it? I personally thought that "File to format" isn't a description of what `--format` does, but rather the description of the value it takes?

Reply:
Hm maybe you're right. Also the output from `--help` looks quite bad with the unaligned columns :(